.test-infra/jenkins/job_LoadTests_Combine_Flink_Python.groovy

```diff
@@ -32,7 +33,7 @@ String pythonHarnessImageTag = "${dockerRegistryRoot}/python:${dockerTag}"
 String flinkVersion = '1.7'
 String flinkDownloadUrl = 'https://archive.apache.org/dist/flink/flink-1.7.0/flink-1.7.0-bin-hadoop28-scala_2.11.tgz'
 
-def loadTestConfigurationsFiveWorkers = { datasetName -> [
+def scenarios = { datasetName -> [
```

Comment:
It seems that `datasetName` is not used. Is that intentional?

Reply:
it is used to fill `metrics_dataset` according to triggering context (pr/not a pr).